KuringApp/KuringApp/Setting/AppIconSelectView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  AppIconSelectView.swift
+//  KuringApp
+//
+//  Created by 박성수 on 2023/09/27.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+enum KuringIcon: String, CaseIterable {
+    case kuring_app
+    case kuring_app_classic
+    case kuring_app_blueprint
+    case kuring_app_sketch
+    
+    var korValue: String {
+        switch self {
+        case .kuring_app: return "쿠링 기본"
+        case .kuring_app_classic: return "쿠링 클래식"
+        case .kuring_app_blueprint: return "쿠링 블루프린트"
+        case .kuring_app_sketch: return "쿠링 스케치"
+        }
+    }
+}
+
+struct AppIconSelect: Reducer {
+    struct State: Equatable {
+        var appIcons: [KuringIcon] = KuringIcon.allCases
```

Comment:
```suggestion
        var appIcons: IdentifiedArrayOf<KuringIcon> = KuringIcon.allCases
```
Index 기반이 아닌 id 기반으로 안전하게 조회할 수 있기 위함